main.sh

```diff
@@ -3,7 +3,8 @@
 ################################
 # Pull in the configuration
 ################################
-source '.env'
+BASEDIR=$(dirname "$0")
+source "$BASEDIR/.env"
```

Comment:
Since this is a bash script (`#!/usr/bin/env bash`), I would just use:

```bash
BASEDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```
(https://stackoverflow.com/questions/59895/getting-the-source-directory-of-a-bash-script-from-within)

That said, maybe there's room for argument for /etc/sd_serverdiagnostics.env.  This would simplify installation on AWS CodeDeploy environments (cc @dborishansky.)  This is because CodeDeploy wants to deploy the config file, but then the dir is not empty to clone the git repo into.